clients/src/main/java/org/apache/kafka/clients/admin/DescribeAclsResult.java

```diff
@@ -19,20 +19,28 @@
 
 import org.apache.kafka.common.KafkaFuture;
 import org.apache.kafka.common.acl.AclBinding;
+import org.apache.kafka.common.acl.AclBindingFilter;
+import org.apache.kafka.common.annotation.InterfaceStability;
 
 import java.util.Collection;
 
 /**
- * The result of the describeAcls call.
+ * The result of the {@link KafkaAdminClient#describeAcls(AclBindingFilter)} call.
+ *
+ * The API of this class is evolving, see {@link AdminClient} for details.
  */
+@InterfaceStability.Evolving
 public class DescribeAclsResult {
     private final KafkaFuture<Collection<AclBinding>> future;
 
     DescribeAclsResult(KafkaFuture<Collection<AclBinding>> future) {
         this.future = future;
     }
 
-    public KafkaFuture<Collection<AclBinding>> all() {
```

Comment:
Door number 3... I think maybe `bindings` is the way to go, for extensibility... :)